apps/frontend/src/components/Inbox/InboxItems.tsx

```diff
@@ -34,11 +41,26 @@ export const InboxItems: React.FC = () => {
     if (!isFetched) {
       fetchInbox()
     }
-  }, [session, fetchInboxData, setIsFetched])
+  }, [isFetched, fetchInbox])
 
-  const handleExpand = (item: InboxItem) => {
-    setSelectedItem(item)
-  }
+  const handleExpand = useCallback(
+    (item: InboxItem) => {
+      setSelectedItem(item)
+    },
+    [setSelectedItem]
+  )
+
+  const handleRightClick = useCallback(
+    (e: React.MouseEvent, item: InboxItem) => {
+      e.preventDefault()
+      setOpenDropdownId(item._id)
```

Comment:
You'll need to add null here in or condition

![image](https://github.com/user-attachments/assets/022dd793-c001-428f-b0c6-bd244299ca8f)